libs/designsystem/src/lib/components/radio/radio.component.scss

```diff
@@ -41,6 +41,11 @@ $spacing-to-label: map-get($checkbox-radio-spacing, 'to-label');
       margin-left: $spacing-to-edge;
       margin-right: $spacing-to-label;
     }
+
+    span {
+      align-self: center;
+      line-height: line-height('n');
```

Comment:
Should the line-height be set on the entire comp instead? 🤔

Reply:
Should be resolved by: a8e1e31